### ISSUE TITLE
What can be learned about the structure when comparing the form factors between experiments and simulations?

### ISSUE THREAD

**Issue by ohsOllila** [NMRLipids/NMRLipids_V-Review#5]
I have thought that if the form factor is reproduced by the simulation, the electron density profile should be reasonable. However, since some people are tuning the peak highs for better agreement, I am not sure. There is also some connection to the thickness. There is already some discussion about this in the blog with Peter Heftberger and Georg Pabst. Any kind of information from full exaplanation with citations to the hints of relevant literature are helpful here.

**Comment by GeorgPabst** [comment]
I did not check the blog recently...
Anyway there are two applications:
(i) MD is used as a guide to model bilayer structure using the scattering density profile (SDP) descriptions, which described membrane structure based on volume probability distributions of quasi-molecular fragments of the constituent lipids. That is MD guides the parsing of the lipid structure. The structure is the optimized against x-ray and neutron scattering data. A combination of SAXS/SANS is useful to achieve highest structural contrast.
(ii) Given that MD reproduces experimental form factors well gives something like a green card to scrutinize MD data, e.g. for molecular tilt of given lipid species etc.

**Comment by ohsOllila** [comment]
Thanks for the clarifications. I think I know nowadays the literature you are referring.

I have also learned now from the literature that the scaling of experimental form factor is ok, since the absolute electron density is not available from experiments.

Follow up question:
If there is a situation that simulated form factor does not match with experiments, for example, the minimas are not matching. Is it possible to immediately deduce from reciprocal plot if the bilayer is, for example, too thin or thick? I now that you can start to run new simulations to get better fit, but this is a lot of work. Alternatively, you could use a simpler model, e.g. SimtoExp program, to extract structural parameters from form factors and compare to simulations to see which direction to go. Is this the simplest way to say anything, or is there anything to be seen intuitively from reciprocal space?

**Comment by GeorgPabst** [comment]
Yes, a shift of the second minimum of the FF (q~0.2-0.3 Å^(-1)) to higher q indicates membrane thinning and vice versa (i.e. when the minimum shifts to lower q would indicate that the simulated membrane is too thick).

**Comment by ohsOllila** [comment]
It is now written in the manuscript that "Neutron experiments typically explore the contrast between hydrogen and deuterium~\cite{Marquardt15}, e.g. SANS on protiated lipid bilayers suspended in 100\% D$_2$O probes mainly the position of the glycerol backbone."

Why do they probe the glycerol backbone position? I would assume that contrast between hydrogen and deuterium in D_2O would be at the interface between hydrophobic and hydrophilic regions. This happens to be close to the glycerol backbone, however, if I got it right it would better to say something like 
"Neutron experiments typically explore the contrast between hydrogen and deuterium~\cite{Marquardt15}, e.g. SANS on protiated lipid bilayers suspended in 100\% D$_2$O probes mainly the hydrophopic region thickness."
Or is there some other reason why they specifically probe the glycerol backbone location?

**Comment by GeorgPabst** [comment]
No specific reason...just as it is close to it. Anyway your suggestion is more precise...i.e. use "hydrophobic thickness"

**Comment by ohsOllila** [comment]
I have a question about Neutron scattering: Are deuterated molecules always used, or can the experiments be done without? 

I think right now we refer to the experiments with D2O, however, (if I have understood correctly) there are also experiments where specific lipid segments are deuterated and the you get very accurate information on these segments. This kind of studies were referred by Markus here: https://github.com/NMRLipids/NMRLipids_V-Review/issues/6

I think that we should shortly mention this issue.

It seems to me that these neutron experiments with specifically labeled deuterons are utilized surprisinly little (or at all) by simulation community, please correct if I am wrong.

**Comment by GeorgPabst** [comment]
There are two ways to change contrast with SANS.
(1) External contrast: Change the H2O/D2O ratio
(2) Internal contrast: Specifically deuterate lipids (e.g. hydrocarbons, head, backbone, or exchange all H for D.

Type (1) is easy and not very expensive, type (2) is expensive (costs for lipids increase by about a factor of 10). I think this explains the difference in number of SANS experiments choosing type 1 or 2. ;-) It's all about money and research being underfunded!
